cmd/dm-ctl/main.go

```diff
@@ -31,24 +31,35 @@ import (
 	"github.com/pingcap/dm/pkg/utils"
 )
 
+func helpUsage(cfg *common.Config) {
```

Comment:
how about adding a comment for example output?

Reply:
add output [example](https://github.com/pingcap/dm/pull/287/files#diff-61075abbd9aac2de49a1fe702632b302R35)